public/js/components/error.jsx

```diff
@@ -1,20 +1,14 @@
-'use strict';
+import React, { Component, PropTypes } from 'react';
+import { gettext } from 'utils';
 
-var React = require('react');
 
-var utils = require('utils');
+export default class ErrorMessage extends Component {
 
-var gettext = utils.gettext;
+  static propTypes = {
+    error: PropTypes.object.isRequired,
+  };
```

Comment:
Ah yeah there's a few of those I reckon. Need to tune up eslint or something to get a handle on the new style.